tests/ui/methods.rs

```diff
@@ -83,6 +83,21 @@ impl T {
     }
 }
 
+pub struct T1;
+
+impl T1 {
+    #![allow(clippy::missing_safety_doc)]
```

Comment:
Can you put this at the top of the file?

Reply:
Done